MANIFEST.in

```diff
@@ -5,4 +5,5 @@ include pygeos/_version.py
 include pygeos/*.dll
```

Comment:
Do you know what's this for?

Reply:
No, I am a bit worried about this too. It should be excluded (auditwheel injects them)

Reply:
Found it. It is a necessity for the WIndows wheelbuilder (auditwheel only works for the Linux/OSX distributions).

See https://github.com/pygeos/pygeos/pull/103

Reply:
(ah, I thought the manifest was only for sdists .. apparently not ;))